NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -1075,6 +1086,34 @@ public synchronized void loadGroupsInvitations(){
 			plugin.getLogger().log(Level.WARNING, "Problem loading all group invitations", e);
 		}
 	}
+	
+	public synchronized void logNameChance(UUID uuid, String oldName, String newName) {
+		try {
+			logNameChange.setString(1, uuid.toString());
+			logNameChange.setString(2, oldName);
+			logNameChange.setString(3, newName);
+			logNameChange.execute();
+		}
+		catch (SQLException e) {
+			e.printStackTrace();
```

Comment:
Any chance we could get a descriptive label here instead of a flat stacktrace? 

Recommendation:

```
plugin.getLogger().log(Level.WARNING, "Failed to log a name change for {0} from {1} -> {2}", new Object[]{uuid, oldName, newName});
plugin.getLogger().log(Level.WARNING, "Exception during change.", e);
```